### PR TITLE
Update docs for basePath custom-routes interop

### PR DESCRIPTION
**Pull request by ijjk**
Follow-up to https://github.com/vercel/next.js/pull/15041 this updates the documentation for custom-routes to mention `basePath` handling with them

### CODE REVIEW

**Review by Timer — DISMISSED** [review]

File: docs/api-reference/next.config.js/headers.md [hunk 1]
```diff
@@ -89,3 +89,37 @@ module.exports = {
   },
 }
 ```
+
+### Headers with basePath support
+
+When leveraging <a href="/docs/api-reference/next.config.js/basepath.md">basePath</a> support with headers each `source` is automatically prefixed with the `basePath` unless you add `basePath: false` to the header.
```
Comment (lfades) [inline docs/api-reference/next.config.js/headers.md:95]:
```suggestion
When leveraging [basePath](/docs/api-reference/next.config.js/basepath.md) support with headers each `source` is automatically prefixed with the `basePath` unless you add `basePath: false` to the header:
```

File: docs/api-reference/next.config.js/redirects.md [hunk 2]
```diff
@@ -65,3 +65,30 @@ module.exports = {
   },
 }
 ```
+
+### Redirects with basePath support
+
+When leveraging <a href="/docs/api-reference/next.config.js/basepath.md">basePath support</a> with redirects each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the redirect.
```
Comment (lfades) [inline docs/api-reference/next.config.js/redirects.md:71]:
```suggestion
When leveraging [`basePath`](/docs/api-reference/next.config.js/basepath.md) with redirects each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the redirect:
```

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 11.3s | 11s | -290ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.826 | 1.811 | -0.02 |
| / avg req/sec | 1368.91 | 1380.34 | +11.43 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.155 | 1.07 | -0.08 |
| /error-in-render avg req/sec | 2164.43 | 2337 | +172.57 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 12.1s | 11.9s | -209ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 9c5447a66d5f3b8ba8c32e39f49551a5eaeb2f64

File: docs/api-reference/next.config.js/rewrites.md [hunk 3]
```diff
@@ -110,3 +110,28 @@ module.exports = {
   },
 }
 ```
+
+### Rewrites with basePath support
+
+When leveraging <a href="/docs/api-reference/next.config.js/basepath.md">basePath support</a> with rewrites each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the rewrite.
```
Comment (lfades) [inline docs/api-reference/next.config.js/rewrites.md:116]:
```suggestion
When leveraging [`basePath` support](/docs/api-reference/next.config.js/basepath.md) with rewrites each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the rewrite:
```

File: docs/api-reference/next.config.js/headers.md [hunk 4]
```diff
@@ -89,3 +89,37 @@ module.exports = {
   },
 }
 ```
+
+### Headers with basePath support
+
+When leveraging [`basePath`](/docs/api-reference/next.config.js/basepath.md) support with headers each `source` is automatically prefixed with the `basePath` unless you add `basePath: false` to the header:
```
Comment (lfades) [inline docs/api-reference/next.config.js/headers.md:95]:
```suggestion
When leveraging [`basePath` support](/docs/api-reference/next.config.js/basepath.md) with headers each `source` is automatically prefixed with the `basePath` unless you add `basePath: false` to the header:
```

File: docs/api-reference/next.config.js/redirects.md [hunk 5]
```diff
@@ -65,3 +65,30 @@ module.exports = {
   },
 }
 ```
+
+### Redirects with basePath support
+
+When leveraging [`basePath`](/docs/api-reference/next.config.js/basepath.md) with redirects each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the redirect:
```
Comment (lfades) [inline docs/api-reference/next.config.js/redirects.md:71]:
```suggestion
When leveraging [`basePath` support](/docs/api-reference/next.config.js/basepath.md) with redirects each `source` and `destination` is automatically prefixed with the `basePath` unless you add `basePath: false` to the redirect:
```

**Review by Timer — APPROVED** [review]

**Review by lfades — APPROVED** [review]
Looks great

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 10.8s | 10.8s | ⚠️ +43ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.844 | 1.909 | ⚠️ +0.06 |
| / avg req/sec | 1356.06 | 1309.34 | ⚠️ -46.72 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.27 | 1.304 | ⚠️ +0.03 |
| /error-in-render avg req/sec | 1969.18 | 1917.27 | ⚠️ -51.91 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 11.9s | 12.1s | ⚠️ +204ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 04a758b9e6b294741437bc28d9ed743418690b7e

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 11.7s | 11.9s | ⚠️ +209ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.139 | 2.06 | -0.08 |
| / avg req/sec | 1168.71 | 1213.4 | +44.69 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.419 | 1.339 | -0.08 |
| /error-in-render avg req/sec | 1761.22 | 1867.71 | +106.49 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 12.7s | 13s | ⚠️ +348ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 18bfb9bd958aff7c74d8a444637fb01550376632

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 11.8s | 11.7s | -83ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.062 | 2.014 | -0.05 |
| / avg req/sec | 1212.23 | 1241.54 | +29.31 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.382 | 1.379 | 0 |
| /error-in-render avg req/sec | 1808.46 | 1812.25 | +3.79 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 962 B | 962 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| buildDuration | 13s | 13.2s | ⚠️ +259ms |
| nodeModulesSize | 66.5 MB | 66.5 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.72 kB | 6.72 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..cead.js gzip | 9.77 kB | 9.77 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.4 kB | 56.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.79 kB | 5.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.17 kB | 6.17 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.9 kB | 51.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 273 B | 273 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js docs/basePath-custom-routes | Change |
| - | - | - | - |
| _error.js | 881 kB | 881 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 881 kB | 881 kB | ✓ |
| link.js | 923 kB | 923 kB | ✓ |
| routerDirect.js | 917 kB | 917 kB | ✓ |
| withRouter.js | 917 kB | 917 kB | ✓ |
| Overall change | 4.53 MB | 4.53 MB | ✓ |

</details>

</details>
Commit: 148ad23634216990f6c9fc96959cef4350934d51

**Pull request merged**